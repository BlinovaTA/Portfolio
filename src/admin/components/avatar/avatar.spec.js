import avatar from "./index";
import { mount } from "@vue/test-utils";

it("runs test correctly", () => {
  const wrapper = mount(avatar);
})