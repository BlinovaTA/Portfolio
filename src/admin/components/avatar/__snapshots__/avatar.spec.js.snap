// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`avatar snapshot 1`] = `
<div
  class="avatar-component"
  style="width: 3rem; height: 3rem;"
>
  <img
    alt="user picture"
    class="image"
  />
</div>
`;
